README.md

```diff
@@ -1,26 +1,61 @@
-# Learn ![elixir logo](http://elixir-lang.org/images/logo/logo.png "Elixir Logo")
+<div align="center">
 
-## Contents
-- [Why?](#why)
-- [What?](#what)
-- [How?](#how)
-- [Learn Elixir](#learn-elixir)
-- [Commands](#commands)
-- [Basic Types](#basic-types)
-- [Functions and Modules](#functions-and-modules)
-- [Generate an Elixir Project](#generating-your-first-elixir-project)
-- [Documentation](#documentation)
-- [Testing](#testing)
-- [Formatting](#formatting )
-- [Publishing](#publishing)
-- [Data Structures](#data-structures)
-  - [Maps](#maps)
-- [Processes](#processes)
-  - [Spawning a process](#spawning-a-process)
-  - [Sending messages between processes](#Sending-messages-between-processes)
-  - [Concurrency](#concurrency)
-- [Further Resources](#further-resources)
+# Learn ![elixir logo](http://elixir-lang.org/images/logo/logo.png "Elixir Logo")
 
+</div>
+
+- [Learn !elixir logo](#learn-)
```

Comment:
@nelsonic just noticed this **after merging**. We shouldn't need this in the Table of Contents, perhaps remove it? 😄 .

Reply:
Yeah, we could remove it in the next iteration. 👌